Runtime/Scripts/Tileset/ParseTileset.cs

```diff
@@ -186,23 +188,37 @@ private static void ReadImplicitTiling(JSONNode rootnode, Tile parentTile)
             {
                 parentTile.contentUri = implicitTilingSettings.subtreeUri.Replace("{level}", parentTile.level.ToString()).Replace("{x}", parentTile.X.ToString()).Replace("{y}", parentTile.Y.ToString());
             }
-            subtreeReader.DownloadSubtree("", implicitTilingSettings,parentTile, test);
+            subtreeReader.DownloadSubtree("", implicitTilingSettings,parentTile, null);
         }
 
-        public static void test(Tile tile)
+        internal static (string[], string[]) GetUsedExtensions(JSONNode rootNode)
+        {
+            var extensionsUsedNode = rootNode["extensionsUsed"].AsArray;
+
+            string[] extensionsUsed = new string[extensionsUsedNode.Count];
+            for (var i = 0; i < extensionsUsedNode.Count; i++)
```

Comment:
Volgens mij ontploft dit wanneer er geen extentionsUsed array bestaat, en die array is niet 'required'
Daar zou dus het scenario nog bij moeten dat de tileset geen extenties heeft, en die hele array daarom niet heeft opgenomen in de tileset.sjon

Reply:
ik kreeg geen errors bij de tilesets die we gebruiken, dus volgens mij gaat het goed, maar ik zal even een expliciete check toevoegen